pages/products/products.yaml

```diff
@@ -19,74 +19,12 @@ properties:
 # In the brands example, the requests were placed on the block that used them, so they are close
 # to where they are used. Both approaches can be used, but it is best to be consistent throughout an app.
 requests:
```

Comment:
Is this comment still relevent, it looks like the brand requests are defined on the page